SPECS/lld/lld.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        LLD is a linker from the LLVM project that is a drop-in replacement for system linkers and runs much faster than them
+Name:           lld
+Version:        12.0.1
+Release:        1%{?dist}
+License:        NCSA
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://clang.llvm.org
+Source0:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/lld-%{version}.src.tar.xz
+Source1:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/libunwind-%{version}.src.tar.xz
```

Comment:
Can we separate out `libunwind` into its own package?

Reply:
Awkwardly we don't need (or even build) `libunwind` here we need a single header file that lives in `libunwind`'s codebase within the wider LLVM project (that being the `mach-o/compact_unwind_encoding.h` file which `lld` uses to merge Mach-O unwind tables).  I played around with different models for building `lld` (like building it from the full `llvm-project` source bundle which is enormous) and settled on this being the least invasive.

This issue with the standalone `lld` build was introduced in `12.0.0` but persists to this day (`14.0.0-dev`).

Reply:
That sounds reasonable to me. If you wouldn't mind, that feels like something that would be great to document in the spec itself.

Reply:
Done.